spec/models/user_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     it 'is not valid with no email' do
       subject.email = nil
-      expect(subject).to_not be_valid
+      expect(subject).not_to be_valid
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)